site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,384 @@
+            loading="lazy">
+        </a>
+      </div>
+      <!-- Burger menu (visible on small screens) -->
+      <button class="navbar-toggler collapsed" type="button" data-bs-toggle="collapse" data-bs-target=".global-header-1"
+        aria-controls="global-header-1.1 global-header-1.2" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <!-- Navbar with links -->
+      <div id="global-header-1.1" class="navbar-collapse collapse me-lg-auto global-header-1">
+        <ul class="navbar-nav">
+          <li class="nav-item"><a class="nav-link active" href="#">Discover</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Shop</a></li>
+        </ul>
+      </div>
+      <!-- Navbar with action icons -->
+      <div id="global-header-1.2" class="navbar-collapse collapse d-sm-flex global-header-1">
+        <ul class="navbar-nav flex-row">
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/5.2/assets/img/boosted-sprite.svg#search"></use>
+              </svg>
+              <span class="visually-hidden">Search</span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/5.2/assets/img/boosted-sprite.svg#buy"></use>
+              </svg>
+              <span class="visually-hidden">Basket</span>
+              <span class="position-relative align-self-start">
+                <span class="badge bg-info rounded-pill position-absolute top-0 fs-6 text-white translate-middle">
+                  1
+                  <span class="visually-hidden">shopping basket items</span>
+                </span>
+              </span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <img src="/docs/5.2/assets/img/navbar-contact.png" width="25" height="25" role="img" alt="User"
+                loading="lazy">
+              <span class="visually-hidden">My account</span>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+</header>
+<!-- Main -->
+<main class="bd-content order-1">
+  <div class="bg-white title-bar">
+    <div class="container-xxl d-block">
+      <div class="row">
+        <div class="col-md-8 mx-auto">
+          <h1 class="display-1">Become a resseler</h1>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-xxl pt-4">
+    <div class="row">
+      <div class="col-md-8 mx-auto">
```

Comment:
The width of the form doesn't seem to be the same as in the page in the DSM at 1440px. Check also with the designers because the global form doesn't seem to be center in their design guidelines.

Reply:
Seen with @CyriaqueBillard, the form is not centered and it's normal (3 columns on the left / 5 columbns for the form / 4 columns on the right).

Reply:
OK so there is an issue in the DSM.
Considering 1440px screens:
* In https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53/i/108261203 there is 130px between the left edge of the screen and the icon
* In https://system.design.orange.com/0c1af118d/p/37609b-global-headers/b/366c91/i/66611060 it is 70px